src/control/cad.js

```diff
@@ -188,30 +188,34 @@ class CadControl extends Control {
     `;
   }
 
+  handleInteractionAdd = (e) => {
```

Comment:
usually we prefer to bind the function in the constructor. I will make th change on master.

Reply:
see https://github.com/geops/openlayers-editor/commit/c01148df3b39fdeb02cc294b69470a20cee4da10